Runtime/LLMUnitySetup.cs

```diff
@@ -402,7 +410,7 @@ static async Task DownloadLibrary()
 
                 // setup LlamaLib in StreamingAssets
                 await DownloadAndExtractInsideDirectory(LlamaLibURL, libraryPath, setupDir);
-
+#if UNITY_ANDROID
```

Comment:
I will modify this function once the build is working, just leave this as it was :)